README.md

```diff
@@ -310,6 +310,45 @@ extension AppDelegate: UNUserNotificationCenterDelegate {
     }
 }
 ```
+When tracking opened push notification, you can also decrement the badge count on the app icon by adding the following code to the `userNotificationCenter:didReceive:withCompletionHandler` method:
```

Comment:
Should we make it slightly more clear when this will decrement? Maybe "When tracking opened push notification (i.e., when a user taps/opens a push notification), you can also..."

Reply:
Hmm. I was just thinking that since this is within the section of "Tracking open events" and the first line reads as - "When a user taps on a push notification,..." it would already be clear since it is within that context but let me know if you think otherwise.

Reply:
Oh that's a good point. I didn't see that we had "when a user taps on a push notification" in the section above. What you have looks good!